test/test_dependencies.py

```diff
@@ -17,6 +17,7 @@
 import docker
 import pytest
 
+from ros_cross_compile.dependencies import DependenciesStage
```

Comment:
i'd like to see a test in here that invokes `__call__`. perhaps wrap one of the existing tests into a `DependenciesStage` to validate its actual functionality